### ISSUE TITLE
Zilla doesn't communicate with bitnami/kafka in Taxi demo

### ISSUE THREAD

**Issue by vordimous** [aklivity/zilla#690]
**Describe the bug**
Zilla (ZILLA_VERSION=0.9.64) is not brokering messages down to kafka in the [Taxi-demo](https://github.com/aklivity/zilla-demos/tree/main/taxi). The same `zilla.yaml` works when pointing to a redpanda instance at the same `kafka:9092` address. A separate `kafka-init` container is successfully connecting and creating topics in both instances. I have checked the repo at the state before adding the redpanda feature and before the previous dependabot version updates and produced the same results.

gRPC messages are on the request topic but are not getting delivered to the response topic:
![image](https://github.com/aklivity/zilla/assets/3012311/74fa6856-33dc-4eaa-b6ff-ea85b2509c3b)

I updated to the previous version `ZILLA_VERSION=0.9.63` and the issue does not exist.

**To Reproduce**
1. Run the [Taxi-demo](https://github.com/aklivity/zilla-demos/tree/main/taxi) with the `./startup.sh` script
2. Observe no messages are getting to kafka from the mqtt simulator and the `/taxi/locations` GET request from the UI never finishes
3. Stop the demo with `./teardown.sh`
4. Run the [Taxi-demo](https://github.com/aklivity/zilla-demos/tree/main/taxi) with the `./startup.sh --redpanda` script
5. Everything works as expected

**Expected behavior**
Taxi Demo should function as stated in README

**Comment by vordimous** [comment]
Tested against lastest (ZILLA_VERSION=0.9.65) and the issue still exists.

**Comment by vordimous** [comment]
I found an additional issue while replacing Zilla with a new instance there was an extra message delivered to the reply topic: 
![image](https://github.com/aklivity/zilla/assets/3012311/7497fb20-8c29-4e7c-a380-615b697db5aa)

**Comment by attilakreiner** [comment]
We have done some analysis on this with @vordimous with various zilla versions to look for regressions and try to narrow down the issue.

It looks like there is more than one issue at play here causing a mix of unexpected behaviors:

1. The issue where curl hangs seems to be a regression from `0.9.63` to `0.9.64`, having been fixed in `0.9.65`. When bumping zilla to `0.9.65` this issue is not present anymore.

2. There is another issue where at some point messages stop being propagated from the `route-requests` topic to the `route-replies` kafka topic. This seems a regression from `0.9.61` -> `0.9.62` and the behavior seems consistent with the other bug described in https://github.com/aklivity/zilla/issues/719 which is already scheduled to be fixed.

At this point I suggest to take the following course of action:
1. Bump zilla version to `0.9.65` in the example now to make the first issue go away.
2. After https://github.com/aklivity/zilla/issues/719 is fixed, let's revisit this if the second issue still persists.

**Comment by jfallows** [comment]
Blocked by #719

**Comment by akrambek** [comment]
I tried with latest release which has https://github.com/aklivity/zilla/issues/719 fix and issue doesn't reproduce.
